modules/settings/settings.php

```diff
@@ -128,6 +128,8 @@ public function settings_publish_guard() {
 			echo '>' . esc_html( $label ) . '</option>';
 		}
 		echo '</select>';
+
+		printf( '<p class="description">%s</p>', esc_html__( 'Require posts to travel through custom statuses before publishing.', 'vip-workflow' ) );
```

Comment:
FYI I also added some description labels to the settings page in this PR:

![image](https://github.com/user-attachments/assets/b8d0b93c-0e5f-4c9e-a4b8-87b263a920ad)